docs/getting-started-monitoring.md

```diff
@@ -292,6 +316,11 @@ After completing the Monitoring tests on minikube, remove all resources.
    helm uninstall scalar-monitoring
    ```
 
+1. Uninstall `loki-stack` from minikube.
+   ```console
+   helm uninstall loki-stack
```

Comment:
```suggestion
   helm uninstall scalar-logging-loki
```
As I described another comment, I think it is better to use more understandable release name that users can know the purpose of deployment is Logging for Scalar products.

Reply:
Thanks for the advice.
I revised it in https://github.com/scalar-labs/helm-charts/pull/87/commits/eb349d4503420bc0cdcf55ac9bef9de68a97ea80